incubator/sparkoperator/templates/spark-operator-rbac.yaml

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2017 Google LLC
```

Comment:
Not sure if this license header should be removed or replaced with one that's more appropriate for this repo.

Reply:
Good point. I have no idea. Feel free to mention/@ anyone you think would know.

Reply:
Let's remove the header for now.